docker-compose.yml

```diff
@@ -474,16 +474,22 @@ services:
 
   pulumi:
     image: grapl/local-pulumi:${TAG:-latest}
-    command: |
-      /bin/bash -c "
-        cd grapl &&
-        pulumi login --local &&
-        pulumi stack init local-grapl --non-interactive &&
-        pulumi up --yes --skip-preview --stack=local-grapl &&
+    entrypoint: ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
```

Comment:
Would it potentially be more resilient to define this in an `ENTRYPOINT` clause in `Dockerfile.pulumi` instead? That way if (for whatever reason) we move away from docker-compose the container image will retain these entrypoint settings.

Reply:
In this case it's probably six-of-one, half-a-dozen of the other. However, we don't use this container outside of `docker-compose`, and the way it's being used is very dependent on how we're doing things in `docker-compose`, so keeping it all here in the compose file seems like a good approach for the time being.

Reply:
Sounds good.